BCH-Wunder-Project-team-4/Wunder.io
# Insights

## Link to ticket:

https://trello.com/c/kCjV8OXW/42-pages-articles-full-width-image-video-option
https://trello.com/c/jDXT3TKN/83-insights-page

## Link to feature environment:

http://localhost:3000/insights

## Changes proposed in this PR:

created a new page with 2 new paragraph types and a new design of Article cards
![Screenshot 2023-11-30 at 0 11 31](https://github.com/BCH-Wunder-Project-team-4/Wunder.io/assets/117649417/07a45258-cdfb-4207-909f-e26e83bb784f)
![Screenshot 2023-11-30 at 0 13 18](https://github.com/BCH-Wunder-Project-team-4/Wunder.io/assets/117649417/b12ffd40-b34a-4ded-9fb9-ba0da309311a)
![Screenshot 2023-11-30 at 0 14 11](https://github.com/BCH-Wunder-Project-team-4/Wunder.io/assets/117649417/cb3eefbf-6df9-4744-b099-3e613a3ddc7b)

## How to test:

Open http://localhost:3000/insights
there should be content if not, open CMS Content - insights- edit. Add Paragraph  full width with 4 images and 3 text (info about picture sizes and length of text will be added later )
add wunderpedia paragraph. Use the header only once for the first paragraph and then ignore it 
Check how responsible it is, it is possible to check the dark mode 
If you going to check Lighthouse..... be ready to see the worst Performance score ever... It is because of GIF size and currently, I'm working on a solution for that 
## Best practices:

<details>
<summary><h3>Accessibility:</h3></summary>
<p>
This project must support WCAG accessibility level AA <em>(edit this according to the requirements of your project)</em>. To ensure this standard is met, remember to:

- Perform automated checks using a tool such as Wave or SiteImprove.
- Test keyboard navigation: are all parts of the UI navigable using only the keyboard? Is the tab order logical? Can popups, menus etc be dismissed with the escape key?
- Test responsiveness, scaling and text reflow.
- Make sure no accessibility issues exist on either desktop or mobile views.
- If you have time, test with a screen 

reader such as VoiceOver (macOS), NVDA (Windows), or Orca (Linux).

Use the [Accessibility Testing Cheat Sheet](https://intra.wunder.io/info/accessibility-group/accessibility-testing-cheat-sheet) for information on how to run these tests.
</p>
</details>
